packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -191,7 +192,7 @@ const websiteEntityPage = (
     </EntityLayout.Route>
 
     <EntityLayout.Route path="/ci-cd" title="CI/CD">
-      {cicdContent}
```

Comment:
I think you should modify `cicdContent` to render the new card rather than replace it.